pom.xml

```diff
@@ -7,6 +7,17 @@
     <groupId>8thlight</groupId>
     <artifactId>tictactoe</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>RELEASE</version>
+        </dependency>
+    </dependencies>
+    <properties>
+        <maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
As it's specified here, you'll only be able to use features from Java 6 and earlier. If you change these to 1.8 you'll get to use those features.